src/backend/app.js

```diff
@@ -21,10 +21,55 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.listen(port, () => {
+  console.log(`Listening on port ${port}`);
+});
+// Respond with all meals in the future (relative to the when datetime)
+app.get("/future-meals", async (req, res) => {
+  const [rows] = await knex.raw(
+    "SELECT * FROM meal WHERE meal.time>DATE(NOW());"
+  );
+  rows.length !== 0
+    ? res.json(rows)
+    : res.status(200).send("there are no meals in the future");
+});
+//Respond with all meals in the past (relative to the when datetime)
+app.get("/past-meals", async (req, res) => {
+  const [rows] = await knex.raw(
+    "SELECT * FROM meal WHERE meal.time< DATE(NOW());"
+  );
+  rows.length !== 0
```

Comment:
if(rows) is more preferred way to check this

Reply:
Thank you for reviewing my hw. I made changes.